### ISSUE TITLE
Html editor fields

### ISSUE THREAD

**Issue by otamiotto** [premieroctet/next-admin#69]
Possibility of using some html editor library (Rich text editor) in fields configuration

**Comment by foyarash** [comment]
Related to #16 

This is on our roadmap after doing the app router support

**Comment by cregourd** [comment]
This is available on [v3.2.6](https://www.npmjs.com/package/@premieroctet/next-admin/v/3.2.6) 🚀
You con now use this richtext editor by adding property `format: richtext-[json | html]` to your fields options. 

However dependencies that allow rich text to work are `optionalDependencies` in next-admin, so you have to add these dependencies to your project 

```sh
yarn add slate-history@^0.100.0 slate-react@^0.101.5 slate@^0.101.5
```

```
"optionalDependencies": {
    "slate": "^0.101.5",
    "slate-history": "^0.100.0",
    "slate-react": "^0.101.5"
  }
```
Feel free to give feedback
